train.py

```diff
@@ -1,39 +1,72 @@
-#!/usr/bin/env python
+# Build a keras neural network and train it
 
-from utils import Data
-import model
-import tensorflow as tf
+import numpy as np
+from keras.models import Sequential
+from keras.layers import Dense, Dropout, Flatten
+from keras.layers import Conv2D
+from keras import optimizers
+from keras import backend as K
+from utils import Screenshot
 
-# Load Training Data
-data = Data()
+# Global variable
+OUT_SHAPE = 5
+INPUT_SHAPE = (Screenshot.IMG_H, Screenshot.IMG_W, Screenshot.IMG_D)
 
-# Start session
-sess = tf.InteractiveSession()
+def customized_loss(y_true, y_pred, loss='euclidean'):
+    # Simply a mean squared error that penalizes large joystick summed values
+    if loss == 'L2':
+        L2_norm_cost = 0.001
+        val = K.mean(K.square((y_pred - y_true)), axis=-1) \
+                    + K.sum(K.square(y_pred), axis=-1)/2 * L2_norm_cost
+    # euclidean distance loss
+    elif loss == 'euclidean':
+        val = K.sqrt(K.sum(K.square(y_pred-y_true), axis=-1))
+    return val
 
-# Learning Functions
-L2NormConst = 0.001
-train_vars = tf.trainable_variables()
-loss = tf.reduce_mean(tf.square(tf.subtract(model.y_, model.y))) + tf.add_n([tf.nn.l2_loss(v) for v in train_vars]) * L2NormConst
-train_step = tf.train.AdamOptimizer(1e-4).minimize(loss)
+def creation_model(keep_prob = 0.8):
+    # create model
```

Comment:
this comment isn't adding any value imo the function name already has us covered